.github/workflows/integration.yml

```diff
@@ -9,9 +9,13 @@ on:
         description: Otomi installation profile
         default: full
         type: string
+      server_providers:
```

Comment:
It would be more convininent if this parameter is the firs one after branch selection. Now it is 3rd one.
<img width="364" alt="image" src="https://github.com/redkubes/otomi-core/assets/17126497/68235e05-c1f8-4026-89af-5df32bf93138">

Reply:
Agree, I'll put it to the top